doc/api/url.md

```diff
@@ -1316,7 +1362,23 @@ changes:
 The `url.format()` method returns a formatted URL string derived from
 `urlObject`.
 
-```js
+```mjs
+import url from 'url';
+url.format({
+  protocol: 'https',
+  hostname: 'example.com',
+  pathname: '/some/path',
+  query: {
+    page: 1,
+    format: 'json'
+  }
+});
+
+// => 'https://example.com/some/path?page=1&format=json'
+```
+
+```cjs
+const url = require('url');
```

Comment:
This describes a legacy API, I'm not sure it's worth providing an ESM example for this considering that users should avoid using those APIs.